.changeset/terrific-apples-run.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fluidframework/tree": minor
+---
+
+Adds garbage collection for repair data so that removed content is not retained forever.
```

Comment:
Is this a mostly internal change? How will it affect current Tree users? Do they need to adapt their code in any way? Are there new errors that they need to watch out for?

cc: @microsoft/fluid-cr-docs

Reply:
Yes, this is mostly an internal change. It shouldn't affect tree users in a visible way but there are some concerns with cross-version collab that we've determined is low risk. The issues should result in certain errors, so I'll make a note of that in the changeset.